src/main/webapp/donation.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>FeedOn - Donation Form</title>
+
+    <!-- Favicon, CSS & JS -->
+    <link rel="icon" href="assets/logo.png" type="image/png" sizes="16x16">
+    <link rel="stylesheet" href="styles/common.css">
+    <link rel="stylesheet" href="styles/donation-form.css">
+    <script src="js/client.js" defer></script>
+
+    <!-- Font Awesome icons CDN -->
+    <script src="https://use.fontawesome.com/4e51920be5.js"></script>
+
+    <!-- Bootstrap CDN -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+</head>
+
+<body>
+
+    <form action="" class="form-container">
+
+        <h2 class="center">MAKE A FOOD DONATION</h2>
+
+        <!-- Name -->
+        <label><b>Restaurant Name</b></label>
+        <input placeholder="Enter Restaurant name" required>
+
+        <!-- Location -->
+        <label><b>Restaurant Location</b></label>
+        <input placeholder="Enter Restaurant location" required>
+
+        <!-- Food Category -->
+        <label><b>Food Category</b></label> Choose multiple options by pressing the command/ctrl key
+        <br>
+        <select class="mdb-select colorful-select dropdown-primary md-form" multiple searchable="Search here..">
+            <option value="" disabled selected>Select food categories</option>
+            <option value="1">Vegetarian</option>
+            <option value="2">Non-Vegetarian</option>
+            <option value="3">Halal</option>
+        </select>
+        <br>
+
+        <!-- Date & Time -->
+        <br>
+        <label><b>Specify a date and time for donation:</b></label>
+        <input type="datetime-local" id="birthdaytime" name="birthdaytime" required>
```

Comment:
Why is the id birthdaytime?

Reply:
Big oops, I fixed this in my consequent PR.